src/linker/Linker.Steps/BodySubstituterStep.cs

```diff
@@ -71,6 +71,7 @@ void ProcessAssemblies (XPathNodeIterator iterator)
 		void ProcessAssembly (AssemblyDefinition assembly, XPathNodeIterator iterator)
 		{
 			ProcessTypes (assembly, iterator.Current.SelectChildren ("type", ""));
+			ProcessResources (assembly, iterator.Current.SelectChildren ("resource", ""));
```

Comment:
This feels a bit more like hack to me for one specific use case. Could you instead add it to a custom step we build for wasm only and remove it there?

Reply:
This is a general approach which allows moving this logic to wasm instead of putting it into the linker where it doesn't really belong.

Reply:
Also, this pass is called substitution pass, but what it really does is it makes modifications to the assembly based on a config file. Removing resources is just another kind of modification. I can add an 'action' attribute with a 'remove' value to make this more explicit.